sway/sway-bar.5.scd

```diff
@@ -61,6 +61,13 @@ Sway allows configuring swaybar in the sway configuration file.
 *binding\_mode\_indicator* yes|no
 	Enable or disable binding mode indicator. Default is _yes_.
 
+*gaps* <amount>|<horizontal> <vertical>|<top> <right> <bottom> <left>
```

Comment:
Shouldn't be 4 arguments at most here?

Reply:
It is?

Reply:
```
<amount>|<horizontal> <vertical>|<top> <right> <bottom> <left>
1                     2                3       4        5
```

Reply:
1 arg: `gaps <amount>`
2 args: `gaps <horizontal> <vertical>`
4 args: `gaps <top> <right> <bottom> <left>`

Should I reword `<amount>` to something like `<all>`?

Reply:
Ah, I see. Maybe we can put spaces around `|` to make it clearer (I thought it was `<amount>|<horizontal>` followed by `vertical>|<top> …`).

And yeah, "all" would be better.